core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -887,12 +887,24 @@ protected function generateButtons($arrRow, $strTable, $arrRootIds=array(), $bln
 
 		$return = '';
 
+		$security = System::getContainer()->get('security.helper');
+		$subject = new DataContainerSubject($strTable, $arrRow['id']);
```

Comment:
I think we need a `rawurldecode()` here, see 5 lines below.
```suggestion
		$subject = new DataContainerSubject($strTable, rawurldecode($arrRow['id']));
```